.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: kind/bug, lifecycle/needs-triage
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```

Comment:
Suggest we use HTML comments for these instructions so they don't show up if people don't replace them:

```
<!-- A clear and concise description of the bug. -->
```

Reply:
👍